### PR TITLE
add(epp)cisco-cts-and-cyberpower-proc

### PR DESCRIPTION
**Pull request by Sims24**
## Description

New Cisco CTS and CyberPower PDU Systems Packs and monitoring procedure 

## Target serie

- [X] 20.04.x
- [X] 20.10.x (master)

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/hardware-devices-cisco-cts-snmp.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Calls-->
+
+| Metric name                                    | Description                                     | Unit |
+|:---------------------------------------------- |:----------------------------------------------- |:---- |
+| calls.total.unknown                            | Calls count having unknown status               |      |
+| calls.total.other                              | Calls count having other status                 |      |
+| calls.total.internal.error                     | Calls count encountering internal error         |      |
```
Comment (garnier-quentin) [inline en/integrations/plugin-packs/procedures/hardware-devices-cisco-cts-snmp.md:28]: Should be '.count' for the metrics ?

File: en/sidebars.json [hunk 2]
```diff
@@ -107,7 +107,7 @@
         "Alerts & Notifications": [
             "alerts-notifications/introduction",
             "alerts-notifications/concepts",
-            "alerts-notifications/resources-status",
+            "alerts-notifications/events-view",
```
Comment (garnier-quentin) [inline en/sidebars.json:110]: Is it ok that change ?

File: en/sidebars.json [hunk 3]
```diff
@@ -181,7 +181,8 @@
                 ],
                 "label": "Administrate",
                 "type": "subcategory"
-            }
+            },
+            "graph-views/release-notes"
```
Comment (garnier-quentin) [inline en/sidebars.json:185]: Is it ok ?

File: fr/integrations/plugin-packs/procedures/hardware-pdu-cyberpower-snmp.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+## Vue d'ensemble
+
+Le Plugin-Pack CyberPower PDU 
+
+## Contenu du Plugin-Pack
+
+### Objets supervisés
+
+* Les statuts des *Périphériques*
+* L'historique des *Appels* et leurs métriques de performances
```
Comment (garnier-quentin) [inline fr/integrations/plugin-packs/procedures/hardware-pdu-cyberpower-snmp.md:15]: Not really. Wrong copy/paste ;)

File: en/integrations/plugin-packs/procedures/hardware-devices-cisco-cts-snmp.md (hunk 1 above)
Comment (Sims24) [reply en/integrations/plugin-packs/procedures/hardware-devices-cisco-cts-snmp.md:28]: Yep !

File: fr/integrations/plugin-packs/procedures/hardware-pdu-cyberpower-snmp.md (hunk 4 above)
Comment (Sims24) [reply fr/integrations/plugin-packs/procedures/hardware-pdu-cyberpower-snmp.md:15]: Erf thanks

File: en/integrations/plugin-packs/procedures/hardware-devices-cisco-cts-snmp.md (hunk 1 above)
Comment (Sims24) [reply en/integrations/plugin-packs/procedures/hardware-devices-cisco-cts-snmp.md:28]: done

File: en/sidebars.json (hunk 2 above)
Comment (Sims24) [reply en/sidebars.json]: Tried to update. Should be ok now.

File: en/sidebars.json (hunk 3 above)
Comment (Sims24) [reply en/sidebars.json]: same

**Review by garnier-quentin — APPROVED** [review]

**Pull request merged**